doc/meetings/build-wg-meeting-2024-05-15.md

```diff
@@ -0,0 +1,56 @@
+# Node.js  Build WorkGroup Meeting 2024-05-15
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=CVmBq_wFd-8
+* **GitHub Issue**: https://github.com/nodejs/build/issues/3716
+* **Minutes Google Doc**: https://docs.google.com/document/d/1PFqf2H5gW12K92QEcoiDjH9Y2l_oFEZ-VOjEhtkZlN8/edit
+
+## Present
+
+* Ulises Gascon: @UlisesGascon
+* Michael Dawson: @mhdawson
+* Richard Lau: @richardlau
+* Moshe Atlow: @MoLow
+
+## Agenda
+
+## Announcements
+
+Github bot is currently not working. There is an issue with the dependencies. It will require some manual work as not everything can be done via the current Ansible scripts. Note that this is a legacy service (Node@14). https://github.com/nodejs/build/issues/3720
+Code static analysis is broken and we have an alternative in place
+We needed to update the Python version on the release machines due the new version of gyp-next.
+
+
+
+*Extracted from **build-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/build
+
+* Infrastructure for Orka (2024 and beyond) [#3686](https://github.com/nodejs/build/issues/3686)
+  * Path has been defined in the issue
+  * Onboarding with Ryan complete, he will take ownership of adding MacOS 13 nodes
+  * Right now our physical arm machines are down and we don’t have any arm testing
```

Comment:
We have arm testing on GitHub actions

Reply:
solved in https://github.com/nodejs/build/pull/3725/commits/435a6d1ee90c5a02b66abe097f0af4d65d2bec0d